src/alert-syslog.c

```diff
@@ -35,9 +35,9 @@
 
 #include "detect.h"
 #include "detect-parse.h"
-#include "detect-engine.h"
-#include "detect-engine-mpm.h"
-#include "detect-reference.h"
+#include "detect/keywords/engine.h"
```

Comment:
`engine` is not a keyword. My idea was to distinguish between keywords and engine internals `src/detect/keywords/*` and `src/detect/engine/*`

Reply:
Fixed.